src/Apps/Order/Components/__tests__/AddressVerificationFlow.jest.tsx

```diff
@@ -96,16 +104,36 @@ describe("AddressVerificationFlow", () => {
       expect(screen.getByText("Edit Address")).toBeInTheDocument()
     })
 
-    it('calls onChosenAddress with the suggested address when "Use This Address" is clicked', async () => {
+    it("tracks that the validation modal was seen", async () => {
       renderWithRelay(
         {
           VerifyAddressType: () => mockResult,
         },
         undefined,
+        componentProps
+      )
+      await screen.findByText("Check your delivery address")
+
+      expect(trackEvent).toHaveBeenCalledTimes(1)
+      expect(trackEvent).toHaveBeenCalledWith({
+        action_type: "validationAddressViewed",
+        context_module: "ordersShipping",
+        context_page_owner_id: undefined,
```

Comment:
This field is set the order ID for other events that we track in the checkout flow. Why is it `undefined` for this event?

Reply:
hey, adding in a comment after merge to say that there are ways to mock out useTracking and force the data you want for testing. using a` trackingSpy` or something else like that in Force.